src/components/AddFundsInterstitial.js

```diff
@@ -185,6 +186,10 @@ const AddFundsInterstitial = ({ network }) => {
           params: !isNaN(amount) ? { amount } : null,
           screen: Routes.ADD_CASH_SCREEN_NAVIGATOR,
         });
+        analytics.track('Tapped Add Cash', {
+          category: 'add cash',
+          newUser: true,
```

Comment:
This isn't technically a new user, but a new Wallet.  For ex. if you create a new wallet it will be empty and you'll see this